src/devices/machine/ncr53c90.h

```diff
@@ -333,9 +345,16 @@ class ncr53cf94_device : public ncr53c94_device
 	ncr53cf94_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
 };
 
+class ncr53cf96_device : public ncr53c94_device
+{
+public:
+	ncr53cf96_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+};
```

Comment:
Is this meant to behave differently somehow, or is it just an alias?

Reply:
It's just an alias for now in MAME but at the chip level there's some extra features in the 53CF96 compared to the 53CF94 (53CF96 supports "differential SCSI bus" whereas 53CF94 is just "single-ended SCSI bus").